src/controllers/todo-controller.ts

```diff
@@ -55,4 +62,34 @@ export class TodoController extends BaseController {
     );
     res.status(201).json(todo.serialize());
   }
+
+  private deleteTodo = async (
+    req: ExtendedRequest,
+    res: Response,
+    next: NextFunction,
+  ) => {
+    const failures: ValidationFailure[] = Validation.extractValidationErrors(
+      req,
+    );
+    if (failures.length > 0) {
+      const valError = new Errors.ValidationError(
+        res.__('DEFAULT_ERRORS.VALIDATION_FAILED'),
+        failures,
+      );
+      return next(valError);
+    }
+
+    const { id } = req.params;
+    
+    const todo = await this.appContext.todoRepository.update(
+      {_id:id, isActive: true},
```

Comment:
Use prettier extension for formatting the file to maintain consistency across the project

Reply:
ok